cpp/src/lance/arrow/file_lance.cc

```diff
@@ -21,6 +21,7 @@
 #include <memory>
 
 #include "lance/arrow/file_lance_ext.h"
+#include "lance/format/format.pb.h"
```

Comment:
where is this used?

Reply:
Removed. it was due to some forward declaration during dev.